backend/public-graph/graph/resolver.go

```diff
@@ -2290,6 +2327,28 @@ func (r *Resolver) AddTrackProperties(ctx context.Context, sessionID int, events
 				return e.New("error deserializing track event properties")
 			}
 
+			attributes := make(map[string]string)
+			for k, v := range propertiesObject {
+				attributes[k] = fmt.Sprintf("%v", v)
+			}
+
+			// make event name the event key and delete from map
+			eventName := attributes["event"]
+			if eventName == "" {
+				eventName = attributes["segment-event"]
+				delete(attributes, "segment-event")
+			} else {
+				delete(attributes, "event")
+			}
+
+			sessionEvents = append(sessionEvents,
+				&clickhouse.SessionEventRow{
+					Event:      eventName,
+					Timestamp:  event.Timestamp,
+					Attributes: attributes,
+				},
+			)
+
```

Comment:
essentially all track events will be recorded, but what about click events? would they have an event name? i'm cool with starting with just the track events, but perhaps we could report other event types as well in the future

Reply:
I'll create a card to tackle the click events - may be a subset of all events that we want to include?